src/app/components/layout/loading-content/loading-content.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-loading-content',
+  templateUrl: './loading-content.component.html',
+  styleUrls: ['./loading-content.component.scss']
+})
+export class LoadingContentComponent {
+  @Input() isLoading = false;
```

Comment:
This should be `false` by default, that way it would not be necessary to assign a value to this property on pages that only use this control to display the load animation (like BlockchainComponent).

Reply:
Fixed.